<launch>
  <!-- load highlight mode params -->
  <rosparam file="$(find ar_star_ros)/params/highlight_mode.yaml" command="load"/>
  
  <!-- Launch the service server -->
  <node pkg="ar_star_ros" type="highlight_solver" name="highlight_solver" output="screen" />

  <!-- Launch the service client -->
  <node pkg="ar_star_ros" type="highlight_listener" name="highlight_listener" output="screen" />
</launch>